app/pages/Settings/FullServiceDirectory.view/FullServiceDirectory.view.tsx

```diff
@@ -55,6 +55,19 @@ const FullServiceDirectory: FC<FullServiceDirectoryProps> = ({
             {`${fullServiceBinariesPath}`}
           </Typography>
         </div>
+        <Box py={1} />
+        <Typography variant="body2" color="textPrimary">
+          Full-Serice logs path
```

Comment:
This text should probably come from the internationalization files

Reply:
Only if we're going to translate the text into another language. I think that contacting a translator is out of scope for this task. If we're not going to provide alternate language translations, I don't think there's a reason to use the translation files.

Reply:
We already have most of the other text coming from translation files, so we should continue to use that and just use English for the languages that we don't currently have translations for yet. It will make it easier to update these in the future.